engines/saga/itedata.cpp

```diff
@@ -590,6 +643,12 @@ const char *optionsStr[][4] = {
 		"\"Si, grazie. Ne avrei bisogno.\"",
 		"\"No, grazie, voglio provare a risolverlo da solo.\"",
 		"Penso che la tessera %s sia nel posto sbagliato."
+	},
+	{
+		"\"Je r\202soudrais cette \202nigme plus tard.\"",
```

Comment:
`résoudrais` is conditional tense, the other languages seem to use future.

Reply:
I think it's good because it say he will do it later, it is future.
Or now he want help to resolve now.

Reply:
Ok, then `résoudrai` is the proper future.